doc/dev/design/FABRID.rst

```diff
@@ -0,0 +1,605 @@
+again by an AS-AS DRKey, that is only known to trusted infrastructure in the source AS.
+
+FABRID policy selection shortcuts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the FABRID policy selection procedure, we could add short cuts for the identifiers, such as:
+``2@REJECT``, to reject all paths that pass through ISD2.
+
+Rationale
+==========
+
+Path type vs HBH extension
+--------------------------------
+
+FABRID can be implemented either as a HBH extension or a path type.
+The reason why we decided against a path type is that FABRID as a HBH extension is incrementally deployable, whereas
+a new path type is not.
+The drawback of this solution is that for the FABRID HBH extension, the Identifier HBH extension, and the HBH extension
+header itself, we need in total 8 additional bytes per path compared to a design that uses a path-type.
+
+Separate Identifier option
+---------------------------
+
+Instead of just having a single FABRID HBH option, we decided to move the packet ID and packet timestamp to another HBH option,
+the so called Packet Identifier Option, because this might also be useful for other HBH extensions and not just for FABRID
+(e.g., it would allow to port EPIC-HP from a path type to a HBH extension).
+Since FABRID still requires the packetID and packet timestamp, providing the Packet Identifier Option became mandatory for FABRID packets.
+The cost of moving the Packet Identifier to a separate HBH extension is 4 bytes, where 2 bytes are used for the HBH option type and length
+fields and 2 bytes for padding to have the FABRID HBH extension 4 bytes alligned.
+
+Length of PacketID and PacketTimestamp for the Identifier HBH option
+---------------------------------------------------------------------
+
+The Identifier has a timestamp with a length of 27 bits, which encodes the relative time in milliseconds after
+the timestamp value of the first InfoField of the SCION header.
+The 27 bit allow to save relative timestamps with a difference of up to 37 hours which fulfills the requirement
+that a path can be valid for up to 24 hours.
+
+Length of FABRID policyID and how to determinte whether policy is local or global
+----------------------------------------------------------------------------------
+
+The decision on whether a certain FABRID policy is a local or global policy is done by the control service,
+hence we do not have to reserve any bits of the FABRID policy index in the FABRID packets to encode whether
+it is a local or global policy.
+In the header design the FABRID policyIndex has a length of 1 byte, which allows 256 different options.
+But since the control service can configure the policies per interface pair and / or per IP range, there
+are many more options than the 256.
+
+
+Compatibility
+===============
+
+FABRID is a new extension which uses the SCION Hop-by-Hop extension which allows
```

Comment:
FABRID uses the SCION ...
Remove "is a new extension which"

Reply:
Done